src/mf/mf-uvc.cpp

```diff
@@ -99,10 +99,15 @@ namespace librealsense
 
                     if (nullptr == pMetadata) // Bail, no data.
                         return false;
-
-                    if (pMetadata->MetadataId != MetadataId_UsbVideoHeader) // Wrong metadata type, bail.
+                    try
+                    {
+                        if (pMetadata->MetadataId != MetadataId_UsbVideoHeader) // Wrong metadata type, bail.
```

Comment:
Also check for `pMetadata` validity

Reply:
The pointer pMetadata has a valid value (!=nullptr), but when accessing the value pMetadata->MetadataId, it throws an exception of accessing forbidden memory.